app/components/related_links/edit_component.html.erb

```diff
@@ -1,2 +1,2 @@
-<%= render Elements::Forms::TextFieldComponent.new(field_name: :text, label: I18n.t('works.edit.fields.related_links.text.label'), form:) %>
-<%= render Elements::Forms::TextFieldComponent.new(field_name: :url, label: I18n.t('works.edit.fields.related_links.url.label'), form:) %>
+<%= render Elements::Forms::TextFieldComponent.new(field_name: :text, label: I18n.t('related_links.edit.fields.text.label'), form:) %>
+<%= render Elements::Forms::TextFieldComponent.new(field_name: :url, label: I18n.t('related_links.edit.fields.url.label'), form:) %>
```

Comment:
Divorce the related links i18n from works, since we will use them as part of collections with the same labels.